modules/gsuite_enabled/gsuite_variables.tf

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
This should be renamed to `variables.tf` to conform with Terraform best practices.